libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -1050,4 +1067,86 @@ export const buildTigerSpecificFunctions = (
             });
         });
     },
+    getAllCSPDirectives: async (): Promise<Array<ICSPDirective>> => {
+        try {
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.getAllEntitiesCspDirectives({});
+                return result.data?.data || [];
+            });
+        } catch (error) {
+            return [];
+        }
+    },
+    getCSPDirective: async (directiveId: string): Promise<ICSPDirective> => {
+        try {
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.getEntityCspDirectives({ id: directiveId });
+                return result.data?.data;
+            });
+        } catch (error: any) {
+            throw convertApiError(error);
+        }
+    },
+    createCSPDirective: async (requestData: ICSPDirective): Promise<ICSPDirective> => {
+        try {
+            const jsonApiCspDirectiveInDocument: JsonApiCspDirectiveInDocument = {
+                data: {
+                    id: requestData.id,
+                    type: JsonApiCspDirectiveInTypeEnum.CSP_DIRECTIVE,
+                    attributes: requestData.attributes,
+                },
+            };
+
+            return await authApiCall(async (sdk) => {
+                const result = await sdk.entities.createEntityCspDirectives({
+                    jsonApiCspDirectiveInDocument,
+                });
+                return result.data?.data;
+            });
+        } catch (error: any) {
+            if (error.response?.status === 400) {
+                const message = error?.response?.data?.detail
+                    ? error?.response?.data?.detail
+                    : "Server error";
+                throw new UnexpectedError(message, error);
+            }
+            throw convertApiError(error);
+        }
+    },
+    updateCSPDirective: async (directiveId: string, requestData: ICSPDirective): Promise<ICSPDirective> => {
+        try {
+            const jsonApiCspDirectiveInDocument: JsonApiCspDirectiveInDocument = {
```

Comment:
Please move lines 1118 ~ 1124 into the callback function (that gets passed into `authApiCall` as a parameter) at line 1127. We only need to declare `jsonApiCspDirectiveInDocument` there, don't we?

Reply:
fixed